diff --git a/flang/CMakeLists.txt b/flang/CMakeLists.txt
index 0bfada476..2cafc8f7f 100644
--- a/flang/CMakeLists.txt
+++ b/flang/CMakeLists.txt
@@ -295,7 +295,7 @@ if(FLANG_STANDALONE_BUILD)
   set(LLVM_LIBRARY_OUTPUT_INTDIR ${CMAKE_BINARY_DIR}/lib)
 endif()
 # Always build tco tool
-set(LLVM_BUILD_TOOLS ON)
+# set(LLVM_BUILD_TOOLS ON)  # nono for CI
 
 include_directories(BEFORE
   ${FLANG_BINARY_DIR}/include
